o-clock.cabal

```
@@ -109,7 +112,7 @@ test-suite readme-test
   build-tool-depends:  markdown-unlit:markdown-unlit
   build-depends:       o-clock
                      , markdown-unlit >= 0.5 && < 0.7
-  ghc-options:         -threaded -rtsopts -with-rtsopts=-N -pgmL markdown-unlit
+  ghc-options:         -threaded -rtsopts -with-rtsopts=-N -pgmL markdown-unlit -fno-warn-orphans
```

```suggestion
  ghc-options:         -threaded -rtsopts -with-rtsopts=-N -pgmL markdown-unlit -Wno-orphans
```

The preferred convention for warning flags is `-W`/`-Wno` instead of `-fwarn-`/`-fno-warn`